REQUIREMENTS.txt

```diff
@@ -1,6 +1,6 @@
 click
 matplotlib
-numpy
+numpy != 1.13.0
```

Comment:
the issue is with skimage.peak_local_max (used by starfish.watershedsegmenter) calling the '-' operator on binary np arrays. the issue has been encountered by others (https://github.com/scikit-image/scikit-image/issues/2681)